collect_app/src/main/java/org/odk/collect/android/views/CustomRangeSlider.java

```diff
@@ -0,0 +1,34 @@
+package org.odk.collect.android.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.android.material.slider.Slider;
+
+import org.jetbrains.annotations.NotNull;
+
+public class CustomRangeSlider extends Slider implements  Slider.OnSliderTouchListener {
+
+    public boolean suppressFlingGesture;
```

Comment:
This should be a private field with a getter right? The problem you have otherwise is that something outside the `CustomRangeSlider` could set `suppressFlingGesture` which wouldn't make any sense - it's a value that only the `CustomRangeSlider` should be able to control (write to). 

Remember to think about encapsulation when adding fields/getters/setters to an object/interface.

Reply:
I think it'd also be good for this have a different name. The `CustomRangeSlider` is just tracking whether it's being swiped or not. It doesn't know what that will be used for. Try and think of a name that describes what this value is tracking. Thinking about where it is used is helpful - "the `RangeWidget` suppresses fling gestures (does not allow scrolling from one question to the other) when the slider X".

Reply:
`CusotmRangeSlider` does not reflect what the slider is doing here. Shall I rename it to `SuppressFlingGestureSlider`?